rosdep/base.yaml

```diff
@@ -3248,6 +3248,12 @@ libqglviewer-qt4-dev:
     xenial: [libqglviewer-dev-qt4]
     yakkety: [libqglviewer-dev-qt4]
     zesty: [libqglviewer-dev-qt4]
+libqglviewer-qt5:
```

Comment:
I think the rosdep key name should be libqglviewer**2**-qt5 to match the Debian package name. @tfoote thoughts?

Reply:
Yeah, that would be good to match too.

Reply:
Done :-)